substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/BundleSupport.java

```diff
@@ -119,10 +142,28 @@ String optionName() {
         }
     }
 
+    enum ExtendedBundleOptions {
+        dry_run,
+        container,
+        dockerfile;
+
+        static ExtendedBundleOptions get(String name) {
+            return ExtendedBundleOptions.valueOf(name.replace('-', '_'));
+        }
+
+        @Override
+        public String toString() {
+            return super.toString().replace('_', '-');
```

Comment:
`s/super.toString()/name()`

Reply:
Addressed in commit https://github.com/oracle/graal/pull/6496/commits/e22646ba54ab57f8bd11e56b1a805c7ef3222bce